src/message-dispatch.js

```diff
@@ -52,11 +52,12 @@ export default class MessageDispatch {
     switch (command) {
       case "fly":
```

Comment:
If flying is disabled globally, this command should be ignored and should not appear in the help text when you type "/"

Reply:
I'm wondering now if maybe if you try to enable fly when it's disabled, we message to the user that fly is disabled on the room at the moment.